docs/dependabot-for-private-packagist.md

```diff
@@ -0,0 +1,59 @@
+# Set up Dependabot with Private Packagist
+##
+
+[Dependabot](https://docs.github.com/en/code-security/getting-started/dependabot-quickstart-guide) informs you about vulnerabilities in the dependencies that you use in your repository and can automatically raise pull requests to keep your dependencies up-to-date.  
+
+This guide explains how to configure and set up Dependabot if you want to use it with [Private Packagist](https://packagist.com).
+
+
+## Enable Dependabot in GitHub
+
+Start in your GitHub repository and go to “Settings”. In the “Security” section of the sidebar go to “Code security and analysis”, where you can enable Dependabot. In the process, you will create a dependabot.yaml [as shown as below](#dependabotyaml). Follow [this guide on GitHub](https://docs.github.com/en/code-security/getting-started/dependabot-quickstart-guide#enabling-dependabot-for-your-repository) for the complete steps to enable Dependabot.
+
+![enable dependabot](/Resources/public/img/docs/articles/dependabot-secret.png)
+
+When you click on configure, the dependabot.yaml will be created for you in the .github folder.
+
+### dependabot.yaml
+
+A minimal Dependabot config file would look like this:
+
+```yaml
+version: 2
+updates:
+  - package-ecosystem: "composer"
+    directory: "/"
+    registries:
+      - private_packagist
+    schedule:
+      interval: "weekly"
+registries:
+  private_packagist:
+    type: composer-repository
+    url: https://repo.packagist.com/acme/
+    username: token
+    password: ${{secrets.PRIVATE_PACKAGIST_PASSWORD}}
```

Comment:
```suggestion
    password: ${{secrets.PRIVATE_PACKAGIST_AUTH_TOKEN}}
```

Reply:
Oh just realized this would mean you'd have to retake the screenshots that show this name :/ I think password is suboptimal here, but I guess I can live with it if you don't want to do the screenshots again XD

Reply:
No problem :)